provider/schema/execution.go

```diff
@@ -80,7 +82,14 @@ func (e ExecutionData) callTableResolve(ctx context.Context, client ClientMeta,
 	res := make(chan interface{})
 	var resolverErr error
 	go func() {
-		defer close(res)
+		defer func() {
+			if r := recover(); r != nil {
+				fmt.Print(string(debug.Stack()))
```

Comment:
I suggest to print to standard error:

    fmt.Fprintf(os.Stderr, "Fetch task exited with panic:\n%s\n", debug.Stack())

Reply:
@irmatov Done. Thanks